terraform/test-observer.tf

```diff
@@ -25,6 +25,13 @@ variable "external_ingress_hostname" {
   default     = "canonical.com"
 }
 
+locals {
+  sentry_dsn_map = {
+    production  = "https://dd931d36e0c24681aaeed6abd312c896@sentry.is.canonical.com//66"
+    staging     = "https://84a48d05b2444e47a7fa176b577bf85a@sentry.is.canonical.com//68",
+  }
```

Comment:
needs a value for development or else `local.sentry_dsn_map[var.environment]` will probably fail (that could just be an empty string)